npm-debug.log

```diff
@@ -0,0 +1,40 @@
+0 info it worked if it ends with ok
```

Comment:
You've accidentally checked in `npm-debug.log`, and it probably makes sense to add this to `.gitignore` so that doesn't happen again in the future.